core/sections/clause_8_ogc-process-description.adoc

```diff
@@ -157,6 +157,26 @@ The following JSON Schema fragment illustrates the use of the `format` key to in
+
+The value of the `dataClasses` parameter is an array of URIs.  Each URI identifies a predefined set of properties or a sub-schema.  Two data resources tagged with the same data class URI value can be assumed to each contain all the properties defined by the data class.  This equivalence allows a server to quickly validate that an input data resource meets the server's expections in terms of the properties available for processing, their types, etc. simply by comparing data class URI values.  If the data class URI of the input data resource matches one of the data class URIs specified in the description of the process input (via the `dataClasses` parameter) then the server can be assured that the process can operate on that data resource.
+
+The `dataClasses` parameter is an array allowing process inputs to be described that can handle a variety of data classes.  As long as the data class URI associated with an input data resource matches at least one of the data class URIs listed in the `dataClasses` array the server can assume that the process can operate on that input data process.
+
+This clause started by introducing the concept of the _Data Class_ in relation to a https://docs.ogc.org/is/17-069r4/17-069r4.html#_collection_[feature collection] but the concept is a general one applying to feature collections, coverages, styles, etc.  For example, a specific data class might be defined to include a geometry property _fenceLine_ of type _polygon_.  Thus, any input feature collection tagged with this data class's URI can be expected to include a _fenceLine_ property and its type can be assumed to be _polygon_.  Similarly, a data class could be defined that identifies a set of bands in a coverage, say R, G, B and NIR.  Any coverages tagged with this data class's URI can thus be assumed to contain these bands.
+
+[NOTE]
+====
+. As defined in <<sc-value-passing>> an input value can be passed to a process by value or by reference.  Whether an input value is passed by-value or by-reference is orthogonal to the concept of the _Data Class_.  In either case the server goes through the same procedure (i.e. comparing data class URIs) to determine whether a specific input value is suitable for processing as per the process description.
```

Comment:
@jerstlouis not sure why we need to single out collection input in this context.  "collection" is just another input type but otherwise it behaves like any other input.  You can pass it by-value by encoding the collection into the execute request as a GeoJSON feature collection (you, of course, would probably not do this normally) or you can pass it by-reference via a URL.  For now let's leave this as is.  If, in the course of migrating collection input from Part 3 to Part 1 we see a need to be explicit about it, I can update this section then.